SimonImanigirimpuhwe/my-brand-website-api
# #1 chore: setting up database

#### What does this PR do?

- Setting up database

#### Description of Task to be completed?

- Install express and mongoose packages 

- Install babel for compilation

- Configure babel

- Connect server to mongoDB

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch `ch-setting-up-database-#1`

- Then run `npm install` to install all dependencies

- Run `npm run dev` to start the server

- Use postman send a `GET` request on `http://localhost:3000`

> You should see a welcoming message `Welcome to Simon-tech website`

#### Any background context you want to provide?

- To be connected to DB you need to check `env-sample` for the credentials settings

- You need also to have mongoDB and node installed in your machine

#### What are the relevant Trello stories?

- [#1](https://trello.com/c/skaiBwGq/1-setting-database)

#### Screenshots (if appropriate)

<img width="1440" alt="Screen Shot 2020-10-13 at 19 20 43" src="https://user-images.githubusercontent.com/54619678/95894554-8f40c780-0d89-11eb-87f7-493c3b8423ed.png">


#### Questions:

- N/A
